test/.setup.js

```diff
@@ -0,0 +1,20 @@
+require('babel-register')();
+
+var jsdom = require('jsdom').jsdom;
+
+var exposedProperties = ['window', 'navigator', 'document'];
+
+global.document = jsdom('');
+global.window = document.defaultView;
+Object.keys(document.defaultView).forEach((property) => {
+  if (typeof global[property] === 'undefined') {
+    exposedProperties.push(property);
+    global[property] = document.defaultView[property];
+  }
+});
```

Comment:
In many ways, I'd prefer to construct a fresh DOM environment for each test.  However, this should okay so long as precautions are taken not to pollute global state within tests.